infra/modules/worklytics-connector-specs/main.tf

```diff
@@ -168,7 +168,11 @@ locals {
         "/api/1.0/workspaces/{ANY_WORKSPACE_ID}/projects",
         "/api/1.0/projects/{ANY_PROJECT_ID}/tasks",
         "/api/1.0/tasks/{ANY_TASK_ID}/stories",
-      ]
+      ],
+      secured_variables : [
+        "ACCESS_TOKEN",
+      ],
+      reserved_concurrent_executions : null
```

Comment:
could have put -1, but prefer here in the definition to be `null`, rather than couple it to AWS semantics.

Reply:
I'd rename it just as `max_concurrent_executions` or something like that, `reserved` is part of AWS and not clear to me from specs perspective